docs/manual/guide/production/ConductR.md

```diff
@@ -26,63 +48,92 @@ The above creates what is known as a "bundle". A bundle is the unit of deploymen
 
 ## Loading and running your services during development
 
-From a development perspective ConductR provides a "sandbox" environment where you can start it and then test your bundle long before it goes to production. ConductR's sandbox also permits you to [debug Lagom services](https://github.com/typesafehub/sbt-conductr-sandbox#debugging-application-in-conductr-sandbox) from your IDE when running within its cluster.
+In the previous [Installing ConductR sandbox](#Installing-ConductR-sandbox) section we have already installed the `conductr-cli`. The CLI can be used on the terminal to communicate with the ConductR cluster. 
 
-To access ConductR's sandbox from within the activator console first added the following sbt plugin to your `project/plugins.sbt`:
+To use the same commands within the activator console add the following sbt plugin to your `project/plugins.sbt`:
```

Comment:
@dotta: Because the `conductr-cli` is a prerequisite of `sbt-conductr-sandbox`. You need to install the `conductr-cli` first. The installation of the conductr-cli is "hidden" on the https://www.lightbend.com/product/conductr/developer page. So I want to make sure here that they have installed the `conductr-cli`.